### DISCUSSION TITLE
Welcome to UPG Discussions!

### DISCUSSION THREAD

**Discussion by AvazbekOlimov** [BroOverbugs/UPG#30] [Announcements]
## 👋 Xush kelibsiz!
   Biz Munozaralardan hamjamiyatimizning boshqa aʼzolari bilan bogʻlanish uchun joy sifatida foydalanamiz. Umid qilamizki, siz:
   * Sizni qiziqtirgan savollarni bering.
   * Fikrlarni baham ko'ring.
   * Boshqa jamoa a'zolari bilan muloqot qiling.
   * Boshqalarga xush kelibsiz va ochiq fikrli. Bu biz hamjamiyat ekanligini unutmang
   birga qurish 💪.

   Boshlash uchun quyida o‘zingiz haqingizda taqdimot bilan izoh qoldiring va bu hamjamiyat bilan nima qilayotganingiz haqida bizga xabar bering.

**Comment by Sardoralgoritm** [comment]
Carochi uka discuss yopildi

**Comment by AvazbekOlimov** [comment]
yaxshi zo'r
